tensorly/backend/core.py

```diff
@@ -592,6 +591,9 @@ def kr(self, matrices, weights=None, mask=None):
                 for i in len(matrices):
                     matrices[i].shape = (n_i, m)
 
+        weights : should be complete by writing description about parameter
+        mask : should be complete by writing description about parameter
+
```

Comment:
We need a real description here not a place-holder, e.g.
```
mask : ndarray a mask to be applied to the final tensor. It should be
        broadcastable to the shape of the final tensor, that is
        ``(U[1].shape[0], ... U[-1].shape[0])``.
weights: 1d array of length the number of columns of the matrix
       optional weigths for each column (useful for Kruskal tensors)
```

Reply:
yeah , you are right. that is my fault.i told you i am new in contributing to open source projects. i will open an issue hereinafter rather than complete its description with unusable content.

Reply:
>  i am new in contributing to open source projects

That's why I left comments with appropriate suggestions :)

> i will open an  issue hereinafter

No need, simply add a commit to this pull request with the changes!